thirdparty/build-thirdparty.sh

```diff
@@ -1006,9 +1009,12 @@ build_avro_c() {
     mkdir -p build
     cd build
     $CMAKE_CMD .. -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} -DCMAKE_INSTALL_LIBDIR=lib64 -DCMAKE_BUILD_TYPE=Release
+    OLD_BUILD_SYSTEM=${BUILD_SYSTEM}
```

Comment:
this is a cmake build, cmake will properly generate cmake or ninja conf, current is fine, no need to force a `make` build.